e2e/features/home-office-exposure.feature

```diff
@@ -125,3 +125,44 @@ Feature: Different Home Office roles have different functionality
     And within the `Respondent documents` collection's first item, I should see `RespondentEvidence.pdf` in the `Document` field
     And within the `Respondent documents` collection's first item, I should see `This is the evidence` in the `Description` field
     And within the `Respondent documents` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+    And I click the `Overview` tab
+    Then I should only see the `homeOffice_awaitAppealSkeletonArgument` case progress image
```

Comment:
May be at a later stage but I am thinking this is a potential candidate to use Examples/refactoring and avoid duplication?

Reply:
Agreed. Apparently there is no Cucumber functionality that allows you to run a feature from another feature. You can't even call a scenario from another scenario. (You could do that in early versions of Cucumber, but it was removed) So not exactly sure the best way to reuse some of this duplicated code in this feature.